Lib/fontmake/__main__.py

```diff
@@ -31,6 +31,9 @@ def main():
     parser.add_argument('--family-name', help='Family name to use for masters,'
                         'and to filter output instances by')
     parser.add_argument('--use-afdko', action='store_true')
+    parser.add_argument('--keep-direction', dest="reverse_direction",
+                        action='store_false', help='Do not reverse contour '
+                        'direction')
```

Comment:
We could do away with the reverse_direction parameter in cu2qu and do winding direction testing there, only reversing if necessary for truetype.